globus_automate_client/cli/flows.py

```diff
@@ -57,6 +60,32 @@ class ActionStatus(str, Enum):
     inactive = "INACTIVE"
 
 
+class FlowInputFormat(str, Enum):
```

Comment:
This seems to clear up something that was confusing me on the use of `Action` part of `ActionInputFormat`. Do we need separate Input or Output format enums for Actions and Flows? I'd think not, so perhaps we move the Input and Output format enums to a common place and just call them `InputFormat` and `OutputFormat`.

Reply:
Right now we do probably need at least separate output format enums, since flows can be displayed as 4 different formats while actions can only be displayed as two. In the future I would suggest creating ouput and input helpers and centralizing all the code and logic within. In applications like this I think it's helpful to channel input and output through single entry and exit points within the code.

Reply:
The input formats could likely be consolidated since they are the same two formats (json and yaml)

Reply:
Consolidating input formats seems like a good approach. Given the amount of enums we're using (shout out @typer), it may make sense to split the enums out into their own `enum.py` file. We may be able to get away with something clever by having an `InputFormat` and `OutputFormat` and then a separate FlowsOutputFormat that can take the extra values (+graphviz and image). I'm also fine with avoiding subclassing and creating a separate output format enum for Flows.

Reply:
Okay with my last commit I think I've addressed these concerns (at least partially) and provided a patter for going forward with the different Enum types and helper function interactions.